python/ray/serve/scripts.py

```diff
@@ -122,7 +123,10 @@ def convert_args_to_dict(args: Tuple[str]) -> Dict[str, str]:
     return args_dict
 
 
-@click.group(help="CLI for managing Serve applications on a Ray cluster.")
+@click.group(
+    help="CLI for managing Serve applications on a Ray cluster.",
+    context_settings=CONTEXT_SETTINGS,
```

Comment:
[Nit] Could you pass in the `CONTEXT_SETTING` dictionary directly instead of using a constant? Afaik, this dictionary is only used in this argument, so I don't think it needs to be stored as a global constant.